WARN invalid config registry="NPM_REGISTRY"

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior
<!-- What is the behavior that currently you experience? -->
Can't find local registry

## Expected Behavior
<!-- What is the behavior that you expect to happen? -->
<!-- Is this a regression? .i.e Did this used to be the behavior at one point?  -->
Should have no error message regarding local registry

## Steps to Reproduce
<!-- Help us help you by making it easy for us to reproduce your issue! -->
Follow guide in publishing to verdaccio, `yarn nx-release 999.9.9 --local`

<!-- Can you reproduce this on https://github.com/nrwl/nx-examples? -->
<!-- If so, open a PR with your changes and link it below. -->
<!-- If not, please provide a minimal Github repo -->
<!-- At the very least, provide as much detail as possible to help us reproduce the issue -->
Clone refresh repo

<!-- Remove this line -->
This issue may not be prioritized if details are not provided to help us reproduce the issue.

### Failure Logs
<!-- Please include any relevant log snippets or files here. -->
```
Publishing to npm registry http://localhost:4873/
Continue? (y/n)y
Publishing @nrwl/angular@10.0.11 --tag latest
npm WARN invalid config registry="NPM_REGISTRY"
npm WARN invalid config Must be a full url with 'http://'
npm ERR! code EPUBLISHCONFLICT
npm ERR! publish fail Cannot publish over existing version.
npm ERR! publish fail Update the 'version' field in package.json and try again.
npm ERR! publish fail
npm ERR! publish fail To automatically increment version numbers, see:
npm ERR! publish fail     npm help version
```

### Environment
<!-- It's important for us to know the context in which you experience this behavior! -->
<!-- Please paste the result of `nx report` below! -->
```
  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 10.0.8
  @nrwl/cypress : Not Found
  @nrwl/eslint-plugin-nx : 10.0.8
  @nrwl/express : Not Found
  @nrwl/jest : 10.0.8
  @nrwl/linter : 10.0.8
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 10.0.8
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 10.0.8
  @nrwl/web : Not Found
  @nrwl/workspace : 10.0.8
  typescript : 3.9.6
```

--- a/scripts/publish.sh
+++ b/scripts/publish.sh
@@ -45,7 +45,7 @@ do
   echo "Publishing ${PACKAGE_NAME}@${VERSION} --tag ${TAG}"
 
   if [ "$LOCALBUILD" = "--local" ]; then
-    npm publish --tag $TAG --access public --registry=NPM_REGISTRY
+    npm publish --tag $TAG --access public --registry=$NPM_REGISTRY
   else
     npm publish --tag $TAG --access public
   fi

